### PR TITLE
[TS] LPS-147960 Hide JournalDisplayPageFormNavigatorEntry for all Global Structures

### PR DESCRIPTION
**Pull request by balazssk**
Hi Team,

Can you please review?
https://issues.liferay.com/browse/LPS-147960

Thanks,
Balázs

### CODE REVIEW

**Comment by balazssk** [comment]
ci:test:relevant

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalEditArticleDisplayContext.java [hunk 1]
```diff
@@ -98,6 +103,20 @@ public JournalEditArticleDisplayContext(
...
+			DDMStructure ddmStructure = getDDMStructure();
+
+			Group group = GroupLocalServiceUtil.getGroup(
+				ddmStructure.getGroupId());
+
+			if (group.isCompany()) {
+				ServiceContext serviceContext =
+					ServiceContextThreadLocal.getServiceContext();
+
+				serviceContext.setAttribute("globalStructure", Boolean.TRUE);
```
Comment (lfbesada) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalEditArticleDisplayContext.java:117]: Adding attributes on the ServiceContext is not a good practice, could you please try another approach?

**Review by lfbesada — CHANGES_REQUESTED** [review]
Hi @balazssk,
Thank you very much for the fix! I'm afraid the chosen approach is not a good practice, could you please try another way?
Thanks!

**Comment by balazssk** [comment]
thanks @lfbesada , please see https://github.com/liferay-echo/liferay-portal/pull/7340 for my changes

**Pull request closed without merging**